src/main/java/net/dv8tion/jda/api/interactions/commands/localization/LocalizationFunction.java

```diff
@@ -83,7 +83,6 @@
  * }
  * </code></pre>
  *
- * @see LocalizationMapper#fromFunction(LocalizationFunction)
```

Comment:
This should probably be replaced by `@see Command#setLocalizationFunction` or whatever it is called.